php/commands/role.php

```diff
@@ -258,6 +262,7 @@ public function reset( $args, $assoc_args ) {
 		global $wp_roles;
 		$all_roles = array_keys( $wp_roles->roles );
 		$preserve_args = $args;
+		$this->dry_run         = \WP_CLI\Utils\get_flag_value( $assoc_args, 'dry-run' );
```

Comment:
Does this need to be a class variable? And, to avoid negative conditionals throughout, could we name it something like `$alter_database`?